src/vs/workbench/parts/debug/test/node/debugAdapter.test.ts

```diff
@@ -8,6 +8,7 @@ import * as paths from 'vs/base/common/paths';
 import * as platform from 'vs/base/common/platform';
 import { Adapter } from 'vs/workbench/parts/debug/node/debugAdapter';
 import { TestConfigurationService } from 'vs/platform/configuration/test/common/testConfigurationService';
+import debug = require('vs/workbench/parts/debug/common/debug');
 
```

Comment:
This is the old style of importing we no longer use, instead we should switch to
`import * as debug from 'vs/workbench/parts/debug/common/debug'`
which should be preferable in new TS versions